src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -186,6 +186,16 @@ public bool IsFunction(string name)
         }
 
         /// <summary>
+        /// Lookup a function by name
+        /// </summary>
+        /// <param name="name">name of function</param>
+        /// <returns>function or null if not found</returns>
+        public FunctionDescriptor LookupFunction(string name)
```

Comment:
nit: suggest renaming to GetFuncitonOrNull (matching the naming pattern we have for GetHttpFunctionOrNull)